python-attr-lookup/src/plang/PythonObject.java

```diff
@@ -51,7 +51,10 @@ public List<PythonObject> getMRO() {
      * result (i.e. it remembers the list buildMRO() returned and keeps returning it).
      */
     protected List<PythonObject> buildMRO() {
-        throw new UnsupportedOperationException("not implemented yet");
+        mro = new ArrayList<PythonObject>();
+        mro.add(this);
+        mro.addAll(type.buildMRO());
```

Comment:
You used getType().getMRO(), which is better because you're using the actual getters. getType() won't be any different from `type` at the moment, but it might change, and using buildMRO() will be slower than getMRO().